### DISCUSSION TITLE
Download details of a Role

### DISCUSSION THREAD

**Discussion by skmony** [directus/directus#22287] [Feature Requests]
### Summary

Download details of a Role like createdAt, createdBy

### Motivation

Suppose we want a summary of role 

### Detailed Design

Should be on user-detail page 

### Must Have:
- A Ablity to download the details of a role
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

NA

### Alternatives

NA

### Adoption Strategy

When they want how a role is relevant to the system

**Comment by br41nslug** [comment]
The details of roles can be downloaded using the API. And you're free to add extra fields with more context as you desire.
